### PR TITLE
make class Parser serializable in order to properly run in blazemeter.

### PR DESCRIPTION
**Pull request by marcosprolo**
Hi Andrey, this is the pull request from master. 
Solving the problem for using the plugin in blazemeter

Could you update the hls plugin in plugin manager?

### CODE REVIEW

File: hls/pom.xml [hunk 1]
```diff
@@ -4,7 +4,11 @@
 	<modelVersion>4.0.0</modelVersion>
 
 	<groupId>com.blazemeter.jmeter</groupId>
+<<<<<<< HEAD
```
Comment (undera) [inline hls/pom.xml]: I assume this is undesired

File: hls/pom_BACKUP_15431.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (undera) [inline hls/pom_BACKUP_15431.xml]: What is this file?

**Comment by undera** [comment]
Please looks at the changes yourself. Remove all irrelevant changes please, now it's completely broken.

**Comment by undera** [comment]
Please review the unit test failures

**Comment by sebastianlorenzo** [comment]
Hi Andrey, 

We need to include a JAR which is not in the maven repository. Should we add this JAR to the GIT repository?

This is the JAR https://github.com/inventit/mqtt-websocket-java

**Comment by undera** [comment]
No need adding it to git. Instead, modify .travis.yml to have before_install actions to download that JAR and install it into local maven.

**Comment by pablobarcelo** [comment]
Hi Andrey,
We have updated the documentation of HTTP/2 and HLS and fixed the problem of HLS in BZ. Could you please update the HLS plugin on Plugin Manager?
FYI, we are going to upload the unit tests of WebSocket plugin in the following weeks.
Please let us know if everything is OK.

**Comment by undera** [comment]
Ok, I will release next version of HLS plugin.

**Pull request merged**